ns1/rest/zones.py

```diff
@@ -21,32 +21,59 @@ class Zones(resource.BaseResource):
         "link",
         "primary_master",
         "tags",
+        "views",
     ]
     BOOL_FIELDS = ["dnssec"]
 
+    ZONEFILE_FIELDS = [
+        "networks",
+        "views",
+    ]
+
     def _buildBody(self, zone, **kwargs):
         body = {}
         body["zone"] = zone
         self._buildStdBody(body, kwargs)
         return body
 
     def import_file(
-        self, zone, zoneFile, callback=None, errback=None, **kwargs
+        self, zone, zoneFile, name=None, callback=None, errback=None, **kwargs
```

Comment:
Similar to the change in `__init__.py`, this reorders the parameters and is a breaking change.